### ISSUE TITLE
Kafka MessageSizeTooLarge doesn't say how big is the message

### ISSUE THREAD

**Issue by mkmik** [influxdata/influxdb_iox#1961]
It would be nice if we knew how big was the message that got rejected by kafka.

```
level=debug msg="error writing lines" e="UnknownDatabaseError { source: WriteBufferWritingError { source: KafkaError (Message production error: MessageSizeTooLarge (Broker: Message size too large)) } }" db_name="DatabaseName(\"844910ece80be8bc_3c0bd4c89186ca89\")" num_lines=5000 span=17910817367319904285 target="influxdb_iox::influxdb_ioxd::http" location="src/influxdb_ioxd/http.rs:534" time=1626101250510166492
```

**Comment by mkmik** [comment]
Fixed in #1981

**Comment by mkmik** [comment]
Fun fact: the "(Broker: Message size too large)" is misleading; it seems to imply that the broker said so, while in our case this happened because the producer (client) hits the maximum request size.
